### ISSUE TITLE
Problems running within Wordpress

### ISSUE THREAD

**Issue by terrorfall** [jylauril/jquery-runner#12]
Getting a console error that states the plugin needs jQuery to run when loading in Wordpress. Have added 

jQuery.noConflict();
(function ($) {

to the top of the script, but no joy. jQuery is working for all other plugins.

**Comment by jylauril** [comment]
jQuery has to be loaded before this library. It tries to access it through window.$, I'm guessing it can't find it.

Can you verify that you actually load jquery BEFORE the runner library?

**Comment by terrorfall** [comment]
I can confirm jQuery has been loaded in before the runner.js

**Comment by jylauril** [comment]
Okay I found the problem. The library was expecting to always have the $ function in the window object. When you run the noConflict, jQuery restores the $ function back to what it was before (this case undefined) and expects you to use it through window.jQuery.

I just added a fix for that. Now it first tries to resolve the jQuery and then Zepto and if neither is found then it falls back to $ (this will allow us to support jQuery-like libraries that implements the same syntax).

I updated the code and I'm adding a new release. Get the new version and try it out.
